src/com/google/enterprise/adaptor/prebuilt/DateFilter.java

```diff
@@ -241,7 +254,8 @@ public static DateFilter create(Map<String, String> cfg) {
       format = ISO_8601_FORMAT;
     }
     log.config("format = " + format);
-    SimpleDateFormat dateFormat = new SimpleDateFormat(format);
+    SimpleDateFormat dateFormat = new SimpleDateFormat(
```

Comment:
This is intentional, so that invalid format strings throw exception in create(), whether you use "date" or "days".